source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -522,7 +539,7 @@ namespace RendererQml
 		//TODO: These color styling should come from css
         //TODO: Add hover effect
         backgroundTag->Property("color", context->GetRGBColor(context->GetConfig()->GetContainerStyles().defaultPalette.backgroundColor));
-        backgroundTag->Property("border.color", Formatter() << input->GetId() << ".activeFocus? 'black' : 'grey'");
+        backgroundTag->Property("border.color", Formatter() << input->GetId() << ".activeFocus? 'black' : '#CCCCCC'");
```

Comment:
Where are we getting this color code from?

Reply:
Also looks like we are using the same color code in may places. Can we declare a const string or something at the top? Easy maintenance.

Reply:
The color code was mentioned in [figma](https://www.figma.com/file/fi0VC0FEmETIHTTWANqMqJ/Native-B%26Cs?node-id=0%3A1). I'll make this change. Thanks